packages/tables/src/Actions/ActionGroup.php

```diff
@@ -21,4 +21,11 @@ public function getActions(): array
 
         return $actions;
     }
+
+    public function button(): static
+    {
+        $this->view('tables::actions.group-button');
+
+        return $this;
+    }
```

Comment:
I believe we should also add the link variant of the action just like regular actions have button, link and icon button.